rascal-textmate-core/.gitignore

```diff
@@ -1 +1,6 @@
-target
\ No newline at end of file
+target
+
+# Only used to run tests:
+node_modules
+package-lock.json
+package.json
```

Comment:
Normally package.json is not in the .gitignore

Reply:
Fixed